.travis.yml

```diff
@@ -1,19 +1,21 @@
+sudo: false
+os:
+  - linux
```

Comment:
I think its overly verbose and doesn't serve any meaningful purpose.  Personally.